features/step_definitions/admin_policy_tagging_steps.rb

```diff
@@ -0,0 +1,8 @@
+Then(/^I can tag it to some policies$/) do
+  select policy_1["title"], from: 'Policies'
+  click_button 'Save'
+  publication = Publication.last
+
+  assert_path admin_publication_path(publication)
+  assert_equal [policy_1["content_id"]], publication.policy_content_ids
+end
```

Comment:
For readers: much discussion happened via another channel, inconclusive, we're going to change this one to use helpers and then form some more structured discussion about house style for testing either via a group meeting or an RFC on the wiki.

This is more important than ever now there's a glut of new and different apps being created by different teams but worked on by everyone.